earthpy/tests/test_plot_draw_legend.py

```diff
@@ -0,0 +1,223 @@
+"""Tests for the plot module"""
+
+import numpy as np
+import pytest
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+
+plt.show = lambda: None
+from matplotlib.colors import ListedColormap
+import earthpy.spatial as es
+
+
+@pytest.fixture
+def listed_cmap():
+    cmap = ListedColormap(["white", "tan", "purple", "springgreen", "darkgreen"])
+    norm = mpl.colors.Normalize(vmin=1, vmax=5)
+    return cmap, norm
+
+
+@pytest.fixture
+def binned_array_3bins():
+    im_arr = np.random.randint(10, size=(15, 15))
+    bins = [-np.inf, 2, 7, np.inf]
+    im_arr_bin = np.digitize(im_arr, bins)
+
+    return bins, im_arr_bin
+
+
+@pytest.fixture
+def binned_array():
+    im_arr = np.random.uniform(-2, 1, (15, 15))
+    bins = [-100, -0.8, -0.2, 0.2, 0.8, np.Inf]
+    im_arr_bin = np.digitize(im_arr, bins)
+    return bins, im_arr_bin
+
+
+@pytest.fixture
+def listed_cmap():
```

Comment:
There is already a fixture called `listed_cmap()` defined above this - maybe a whoopsie? The first one returns a tuple, and this returns just one thing. It looks like that's whats causing problems below @lwasser - maybe if we stick with the first `listed_cmap` instead of redefining it here, the tuples below should be unpacked as expected?